build.gradle.kts

```diff
@@ -133,6 +133,24 @@ tasks {
         }
     }
 
+    val createOpenApiSourceJar by registering(Jar::class) {
```

Comment:
What is this needed for?

Reply:
Bundling the source into the plugin, so that consumers of the extension point can compile against the interfaces we provide.

AIUI anyone who implements the extension will need this, per this guide: https://plugins.jetbrains.com/docs/intellij/bundling-plugin-openapi-sources.html#bundling-api-sources-in-gradle-build-script